db/migrate/20221030180046_create_users.rb

```diff
@@ -0,0 +1,10 @@
+class CreateUsers < ActiveRecord::Migration[7.0]
+  def change
+    change_table :users do |t|
+      t.string :name
+      t.string :email
+
```

Comment:
Are we saving passwords too at some point? Or is email enough? Should we implement an email sign in link like slack, linkedin, etc. We could pass a jwt through the email which would act as authorization in a session variable. Or maybe we can use something like the devise gem (seen it pop up in a lot of rails posts on authorization) https://github.com/heartcombo/devise

Reply:
Oh also, missed this in the first pas through. Will we be adding roles for users now or later?

Reply:
We added roles in the roles table as an integer identifier, but I don't think that needs to be part of the user. As far as passwords, I think that is something we can do further down the line if we decide. We were just trying to add the key columns that would be necessary to implement basic features.

Reply:
Ah gotcha, will resolve this then. Thanks for clarifying!